autorest/adal/token.go

```diff
@@ -634,6 +635,31 @@ func GetMSIVMEndpoint() (string, error) {
 	return msiEndpoint, nil
 }
 
+func isAppService() bool {
+	_, endpointEnvExists := os.LookupEnv("MSI_ENDPOINT")
```

Comment:
Can you make MSI_ENDPOINT and MSI_SECRET unexported constants to avoid the duplication.

Reply:
I moved these to constants and replaced all the static string use.